priv/templates/phx.gen.release/dockerignore.eex

```diff
@@ -1,18 +1,25 @@
 .dockerignore
-# there are valid reasons to keep the .git, namely so that you can get the
+# There are valid reasons to keep .git, namely so that you can get the
 # current commit hash
-#.git
-.log
-tmp
+#/.git/
+/.log/
```

Comment:
I know this was already here, but do we know what's generating the `.log` folder? This isn't in the default `.gitignore` files for mix or phoenix projects. I'm just wondering if this line could be removed.

Reply:
It might have been a typo. Maybe `*.log` instead of `.log`. I also understood it's a directory instead of a file.
@chrismccord, can you please help? :)